code/modules/admin/topic/topic_events.dm

```diff
@@ -92,6 +92,12 @@
 			if(amount != 0) //can add negative numbers too!
 				message_admins("[key_name_admin(usr)] added [amount] research credits.")
 				GLOB.chemical_data.update_credits(amount)
+		if("reroll_contracts")
+			var/confirm = tgui_alert(usr, "This will immediatly reroll the contract chemicals, Confirm?", "Reroll Contracts", list("Yes", "No"), 30 SECONDS)
```

Comment:
```suggestion
			var/confirm = tgui_alert(usr, "This will immediately reroll the contract chemicals, Confirm?", "Reroll Contracts", list("Yes", "No"), 30 SECONDS)
```